server/broadcast.go

```diff
@@ -226,7 +226,7 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) {
 	}()
 }
 
-var sessionErrStrings = []string{"dial tcp", "unexpected EOF", core.ErrOrchBusy.Error()}
+var sessionErrStrings = []string{"dial tcp", "unexpected EOF", core.ErrOrchBusy.Error(), core.ErrOrchCap.Error()}
```

Comment:
Should we update `TestStopSessionErrors()` in `broadcast_test.go` to check for this new error?

Reply:
It's actually already updated. The test is designed such that it'll fail if we add a new error value but forget to update the test 😄 See:


https://github.com/livepeer/go-livepeer/pull/729/commits/70342afc9cdd8599241271aa2b0066c23577ddd7#diff-f986f75ea9a0ee09429050246648e0b7R17

Reply:
Oh weird I don't know how I missed that...oops 😓